compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/scene/ComposeSceneMediator.uikit.kt

```diff
@@ -156,6 +156,14 @@ private class SemanticsOwnerListenerImpl(
             current.second.onSemanticsChange()
         }
     }
+
+    override fun onLayoutChange(semanticsOwner: SemanticsOwner, semanticsNodeId: Int) {
+        val current = current ?: return
+
+        if (current.first == semanticsOwner) {
+            current.second.onLayoutChange(nodeId = semanticsNodeId)
+        }
```

Comment:
```suggestion
        val (_, accessibilityMediator) = current?.takeIf { it.first == semanticsOwner } ?: return
        accessibilityMediator.onLayoutChange(nodeId = semanticsNodeId)
```

Reply:
I agree, but I'll leave it to @elijah-semyonov to make this change in a separate PR, if he wants to. I was just copying the style from the function above.